packages/dx-react-grid/src/components/table-layout.test.tsx

```diff
@@ -231,6 +232,19 @@ describe('TableLayout', () => {
         expect(getAnimations)
           .toHaveBeenLastCalledWith(expect.anything(), expect.anything(), 400, new Map());
       });
+
+      it('should work with immutable state', () => {
```

Comment:
You are using the 'state' word in a test name. Actually, you are using immutable array as a property.